### ISSUE TITLE
Speed up the release process

### ISSUE THREAD

**Issue by aminya** [Pure-D/serve-d#134]
Serve-d seems to follow SEMVER, but it doesn't actually do that in reality. The jump from 0.6 to 0.7 has taken more than 500 days. 

Semver versioning suggests that you should bump the minor version for "each" new feature, but in reality, many features are being added for this small jump. I suggest not using SEMVER for this project. The suitable method seems to be one release every two years.

**Comment by WebFreak001** [comment]
I agree the releases are slow, but they are following semver still. The minor version bumps are for backwards-compatible changes, but for 0.x.y versions every minor change is like major versions after a 1.0.0 release and may include backwards-incompatible changes.

I'll consider using things like the Projects board on GitHub to better plan releases, however in the past it used to not work so well. Currently for the next release I only planned to finish the testing implementation, however because of both new test provider APIs being implemented in vscode and also new bugs (esp. memory leakage) being introduced by the partial implementation right now I have been holding onto that for a little longer.

**Comment by aminya** [comment]
We should probably decouple the releases of serve-d from code-d. Many other editors rely on serve-d for their functionality. The bugs in the VsCode should not affect the others.

**Comment by WebFreak001** [comment]
that's a good point. Do you think fixed monthly releases with deadline would help?
